pkg/system/phase4_configuring.go

```diff
@@ -460,6 +467,11 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 		if err := r.prepareGCPBackingStore(); err != nil {
 			return err
 		}
+	} else if r.IsIBMCloud {
```

Comment:
Following up on this flag - why not use the same model as the other providers and just check if `r. IBMCloudCOSCreds.UID != ""` ?
Is the COS flow so different than the other cases?

Reply:
In other clouds, we are check that we've created the cloud operator credentials request CR.
In this case, we aren't creating a CR, we can obviously check if the secret exists or not and assume that it won't exist on other deployments, or check that we've created the IBM secret in our Phase2 (if we want to stick with the current code that has a "useless" secret in the reconciler)